cmd/conf/chain.go

```diff
@@ -52,27 +57,44 @@ func (c *ParentChainConfig) Validate() error {
 	return c.Connection.Validate()
 }
 
+// lint:require-exhaustive-initialization
 type L2Config struct {
-	ID        uint64                   `koanf:"id"`
-	Name      string                   `koanf:"name"`
-	InfoFiles []string                 `koanf:"info-files"`
-	InfoJson  string                   `koanf:"info-json"`
-	DevWallet genericconf.WalletConfig `koanf:"dev-wallet"`
+	ID               uint64                   `koanf:"id"`
+	Name             string                   `koanf:"name"`
+	InfoFiles        []string                 `koanf:"info-files"`
+	InfoJson         string                   `koanf:"info-json"`
+	InitialL1BaseFee string                   `koanf:"initial-l1base-fee"`
+	DevWallet        genericconf.WalletConfig `koanf:"dev-wallet"`
+}
+
+func (c *L2Config) InitialL1BaseFeeParsed() (*big.Int, error) {
+	if c.InitialL1BaseFee == "" {
+		return params.DefaultInitialL1BaseFee, nil
+	}
+
+	parsed, success := big.NewInt(0).SetString(c.InitialL1BaseFee, 10)
+	if !success {
+		log.Error("Failed to parse L1 BaseFee for L2 config (passed with `initial-l1base-fee`)", "config", c.InitialL1BaseFee)
```

Comment:
No need to add an extra log here, error will be logged twice since the error returned by this function will also be logged down the line.
It is just a matter of returning an error with a more descriptive message, like the one in this log line.